src/lib/DataSourceDefinition/BigQuery.ts

```diff
@@ -52,18 +52,17 @@ export default class BigQuery extends Base {
     });
   }
 
-  cancel() {
+  cancel(): void {
     return this._cancel && this._cancel();
   }
 
-  async connectionTest() {
+  async connectionTest(): Promise<void> {
     await bigquery(this.config).query("select 1");
-    return true;
   }
 
-  async fetchTables() {
```

Comment:
If we update `@google-cloud/bigquery` to latest, we can use its type definition for typescript.
https://github.com/googleapis/nodejs-bigquery